rma_reason_code/models/reason_code.py

```diff
@@ -13,8 +13,8 @@ def _get_default_color(self):
         return randint(1, 11)
 
     name = fields.Char("Code", required=True)
-    description = fields.Text("Description")
-    type = fields.Selection(
+    description = fields.Text()
+    rma_type = fields.Selection(
```

Comment:
add explicitly the string="RMA Type" so use uppercase in RMA.

Reply:
done!